niri-config/src/lib.rs

```diff
@@ -745,6 +747,7 @@ pub enum Action {
     FocusWorkspaceDown,
     FocusWorkspaceUp,
     FocusWorkspace(#[knuffel(argument)] u8),
+    FocusWorkspaceBackAndForth,
```

Comment:
Maybe something like `FocusLastWorkspace`?

Reply:
I feel like `FocusPreviousWorkspace` would be more clear since last workspace could be mistaken for the bottom most workspace.